prog/rotate_storage_kek.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+require "shellwords"
+require "json"
+
+class Prog::RotateStorageKek < Prog::Base
+  subject_is :vm_storage_volume
+
+  def start
+    if vm_storage_volume.key_encryption_key_1_id.nil?
+      pop "storage volume is not encrypted"
+    end
+
+    key_wrapping_algorithm = "aes-256-gcm"
+    cipher = OpenSSL::Cipher.new(key_wrapping_algorithm)
+    key_wrapping_key = cipher.random_key
+    key_wrapping_iv = cipher.random_iv
+    auth_data = "Ubicloud-Storage-Auth"
```

Comment:
should auth_data be static or, like....the volume's ULID? (though, I want to change from ulids before it becomes unchangable)